fixtures/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10.14.2-alpine
+
+WORKDIR /opt/cosmosdb
+
+RUN npm install -g ts-node typescript && \
+    npm install documentdb @types/documentdb io-functions-commons \
+                io-ts@1.8.5 fp-ts@1.12.0 italia-ts-commons
```

Comment:
probably we'll need to upgrade to fp-ts@1.17.4. can you check if the fixtures container compiles now I've upgraded some deps on io-functions-commons  and italia-ts-commons, probably we should set a version ie. io-functions-commons@8.0.0 when installing (same for italia-ts-commons)